MetaMask/metamask-extension
# Add obfuscation toggle for private key input

Fixes: https://github.com/MetaMask/metamask-extension/issues/20630

## Explanation
Adds a show/hide element which toggles whether the private key input value is visible or not.

## Screenshots/Screencaps

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/d0b2da48-fed9-403f-aaf3-907b4336b136

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/4d8608fc-f765-4b45-a89b-edb13bfd9542

## Manual Testing Steps
1. Click the accounts menu dropdown
2. Select `Import account`
3. Ensure the private key input defaults to hidden
4. Ensure the show/hide toggle switches between input visibility appropriately
5. Ensure that the private key import functionality is otherwise unchanged

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
